src/test/java/org/folio/rest/impl/TransactionTest.java

```diff
@@ -274,8 +273,9 @@ void testCreateTransferThatDoesNotHaveEnoughMoney() throws MalformedURLException
 
     // create Transfer
     postData(TRANSACTION_ENDPOINT, transactionSample, TRANSACTION_TENANT_HEADER).then()
-      .statusCode(400)
-      .body(containsString(TRANSFER_NOT_ENOUGH_MONEY_ERROR_TEXT));
+      .statusCode(201)
```

Comment:
Need to assert negative available ammount from response payload

Reply:
ok, I will add these cases also